applications/nrf_desktop/Kconfig.ble

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Bluetooth LE configuration"
+
+config DESKTOP_BT
+	bool
+	select BT
+	select BT_SETTINGS
+	select BT_SIGNING
+	select BT_SMP
+
+config DESKTOP_BT_PERIPHERAL
+	bool "Bluetooth LE Peripheral"
+	default y
+	depends on DESKTOP_ROLE_HID_PERIPHERAL
+	select DESKTOP_BT
+	select BT_PERIPHERAL
+
+if DESKTOP_BT_PERIPHERAL
+
+config BT_DATA_LEN_UPDATE
+	default n
+
+config BT_AUTO_PHY_UPDATE
+	default n
+
+config BT_PERIPHERAL_PREF_MIN_INT
+	default 6
+
+config BT_PERIPHERAL_PREF_MAX_INT
+	default 6
+
+config BT_PERIPHERAL_PREF_LATENCY
+	default 99
+
+config BT_PERIPHERAL_PREF_TIMEOUT
+	default 400
+
+config BT_CONN_PARAM_UPDATE_TIMEOUT
```

Comment:
Please remove it also from `nrf52kbd_nrf52832/prj_release_fast_pair.conf` since we update it globally.